### DISCUSSION TITLE
Pc Miner Reset

### DISCUSSION THREAD

**Discussion by Krayze** [revoxhere/duino-coin#449] [Q&A] [answered]
Hi.. is there anyway to reset the miner so I can rename the device, reset intensity levels etc.. as soon as I restart the console it automatically continues to mine with the settings I input.. thanks you

**Comment by revoxhere** [comment, marked as answer]
You can either manually edit Miner_config.cfg or remove the PCMiner_2.45_resources folder and restart the miner to get the configuration tool.

**Reply by Krayze** [reply]
Amazing.. thank you so much.. sorted.. just deleted the resource folder..

**Comment by frisinason** [comment]
[You](https://www.mycfavisit.org/) can manually edit the Miner_config.cfg or delete the PCMiner_2.45_resources folder and restart the miner to get the configuration tool.
